meson.build

```diff
@@ -679,6 +679,10 @@ if time_epoch == -1
         source_date_epoch = run_command('sh', ['-c', 'echo "$SOURCE_DATE_EPOCH"']).stdout().strip()
         if source_date_epoch != ''
                 time_epoch = source_date_epoch.to_int()
+        elif run_command('[', '-d', '.git', ']').returncode() == 0
```

Comment:
not that it matters, but i'd always have used `test` here instead of `[`, simply because it is less cryptic...

Reply:
Good catch, I just googled for how to check if a directory exists in meson and this came up. test is indeed less cryptic.